components/core/src/glt/Grep.cpp

```
@@ -415,6 +562,31 @@
+        } else if (is_wildcard(next_char)) {
+            // if it is an escape followed by a wildcard, we know no escape has been added.
+            // we also remove the original escape because it was purely for query
+            token += next_char;
+        } else if (ir::is_variable_placeholder(next_char)) {
+            // If we are at here, it means we are in the middle of processing a '\\\v' sequence
+            // in this case, since we removed only one escape from the previous '\\' sequence
+            // we need to remove another escape here.
+            token += next_char;
+        } else {
+            printf("Unexpected\n");
+            throw;
```

cppcheck

[error] Rethrowing current exception with 'throw;', it seems there is no current exception to rethrow. If there is no current exception this calls std

(rethrowNoCurrentException)